config/vufind/KohaRest.ini

```diff
@@ -177,3 +177,9 @@ extraFields = item-issue:acceptTerms:pickUpLocation
 ; to disable it:
 ;purgeAll = false
 
+; This section allows libraries to define custom renewal block messages for specific item types.
+; The first string is the item type. In the below example, ILL.
+; The second string after the equal sign is the Koha returned renewal block reason
+; A colon delineates it from the renewal block message to display in VuFind.
+[ItypeRenewalBlockMappings]
+; ILL ="too_many:ill_message"
```

Comment:
Add a space after the equals sign.

Reply:
I think that ILL here should be an array to allow for multiple mappings, e.g.

```
ILL[too_many] = "ILSMessages::ill_item_renewal_limit"
ILL[too_soon] = "ILSMessages::ill_renewal_too_soon"

```
Note that the above also uses the ILSMessages text domain to align the examples with renewalBlockMappings. Using an array also conveniently avoids the confusing colon as a delimiter for both.